src/components/hero/AcceptDemoInvitation.tsx

```diff
@@ -66,6 +71,7 @@ function AcceptDemoInvitation({
                             void mutateAuthRoles();
                         }
 
+                        setHasDemoAccess(true);
```

Comment:
Instead of firing off a full mutate decided just to update the store here. I _might_ like this plan in the future and might remove it. For now I think it is best to reduce calls.